package.json

```diff
@@ -57,5 +58,8 @@
     "readable-stream": "^2.3.7",
     "webpack": "^4.44.2",
     "webpack-cli": "^3.3.10"
+  },
+  "resolutions": {
+    "stream-browserify": "^3.0.0"
```

Comment:
needed for `stream.pipeline` since webpack v4 uses v2 still. Another reason for us to get up to date with webpack